src/components/project-overview/project-overview.tsx

```diff
@@ -6,7 +6,11 @@ import { Project as ProjectInterface } from '../../app/dataModels/project';
 import './project-overview.scss';
 
 export default async function ProjectOverview() {
-  const localProjectData = await getLocalProjectData();
+  let localProjectData;
+  if (!process.env.apiCall) {
+    localProjectData = await getLocalProjectData();
+  }
+
   return (
     <Box className="project-overview-container">
       {localProjectData.map((project: ProjectInterface) => (
```

Comment:
You are right. Because of that, I removed the "else" statement.

Reply:
> You are right. Because of that, I removed the "else" statement.

I meant if `apiCall` is true, then `localProjectData` is not initialized -> resulting error in `return` statement.
Maybe you can change it to 
```
return (
   {!process.env.apiCall && (
     <Box>
     //remaining code
     </Box>
   )}
)
```
?

Reply:
I see. I will do the changes.

Reply:
@nam-m 

I did the required changes.